### DISCUSSION TITLE
Enabling access to Nuxt application via external links

### DISCUSSION THREAD

**Discussion by miraagee** [nuxt/nuxt#27268] [Questions]
Hello!

I'm relatively new to Nuxt and I haven't quite figured the router system out yet. I am sending a link to the user to reset his password, with a hash. I know about how to use pages that can take route-params. The problem is, that, every external access to another page except the first (login) page is being blocked. I can only navigate via code (navigateTo etc..).
It would be great if someone could help.

Best regards, Ben

**Comment by b-mounir-dev** [comment]
@miraagee Did you test with this code (`external` attr/prop is added):
```vue
<NuxtLink to="https://example.com" external>
  Go to external link
</NuxtLink>
```

**Reply by miraagee** [reply]
Hey, thanks for the quick reply. I might've described my problem wrong. I want to enable access to my application via a link I'm sending per mail. So I want to send a link e.g. "www.example-application.com/reset-passwort/12345678". But the user will always be redirected to "www.example-application.com", so the root page. (My root page is [[login.vue]]).

**Reply by b-mounir-dev** [reply]
Do you have a middleware with redirect ?

**Reply by b-mounir-dev** [reply]
@miraagee If you could provide a reproduction code, github repo or stackblitz workspace, would be great!

**Reply by miraagee** [reply]
I actually dont' have any middleware, that's why I can't offer any code. Repo is also private. Just think of a new Nuxt project with 2 pages. Just the first 'landing' page is the one accessible per URL. The other can just be reached with a NuxtLink.

**Reply by TheAlexLichter** [reply]
Are you using the Supabase module? There it is the default to redirect IIRC

**Reply by miraagee** [reply]
Not using any modules for the router..

**Reply by b-mounir-dev** [reply]
> Not using any modules for the router..

Then, you must provide a minimal reproduction code to help you. We can't fix a bug that we can't reproduce! Because we can access nuxt app via link. Example nuxt.com is a nuxt app and you can access any page via its link
